core/server/src/test/java/alluxio/master/file/options/CreateDirectoryOptionsTest.java

```diff
@@ -29,6 +29,7 @@
  * Unit tests for {@link CreateDirectoryOptions}.
  */
 @RunWith(PowerMockRunner.class)
+// Need to mock PermissionStatus to use CommonTestUtils#testEquals.
 @PrepareForTest(PermissionStatus.class)
```

Comment:
the javadoc for testEquals mentions this - basically `@PrepareForTest` is used as a last resort if (1) the field isn't a primitive type (2) the field isn't an enum type (3) the field type is final (4) the field type doesn't have a no-arg constructor. Unfortunately, these happen to all be true for `PermissionStatus`.